nomination/templates/nomination/project_about.html

```diff
@@ -1,59 +1,107 @@
-                      <span class="browser-heading">Internet Explorer</span>
-                      <p class="browser-p">If your Links Toolbar is not visible, right click on the menu bar and select "Links." Right click on this link <a href="javascript:window.open('http://{{current_host}}/nomination/{{project.project_slug}}/url/'+escape(location.href.replace(/\/$/, '')+'/')).focus();">{{project.project_slug}} Nominator</a>, and select "Add to Favorites..." For the "Create in" field, select "Links" then click "Add."</p>
-                  </div>
-              </div>
-              {% endif %} 
-          </div>
-          
-          <div id="about-project-details" class="yui-g">
-              <h3>Project Name:</h3>
-              <p>{{ project.project_name}}</p>
-              <h3>Project Abbreviation:</h3>
-              <p>{{ project.project_slug}}</p>
-              <h3>Project Starting Date:</h3>
-              <p>{{ project.project_start|date:"M d, Y"}}</p>
-              <h3>Project Ending Date:</h3>
-              <p>{{ project.project_end|date:"M d, Y"}}</p>
-              <h3>Nomination Starting Date:</h3>
-              <p>{{ project.nomination_start|date:"M d, Y"}}</p>
-              <h3>Nomination Ending Date:</h3>
-              <p>{{ project.nomination_end|date:"M d, Y"}}</p>
-              <h3>Number of URLs Nominated:</h3>
-              <p>{{ url_count }}</p>
-              <h3>Number of Nominators:</h3>
-              <p>{{ nominator_count }}</p>
-              <h3>Admin Name:</h3>
-              <p>{{ project.admin_name}}</p>
-              <h3>Admin E-mail:</h3>
-              <p>{{ project.admin_email}}</p>
-              <h3>Project URL:</h3>
-              <p>{{ project.project_url}}</p>
-              <h3>Registration Required for Project:</h3>
-              <p>{{ project.registration_required|yesno:"Yes,No" }}</p>
-          </div>
+    <div class="panel-group">
+        {% include "nomination/proj_navbar.html" with active_tab='proj_about' %}
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                <h4 class="panel-title">Project Description</h4>
+            </div>
+            <div class="panel-body">
+                {{ project.project_description|markdown }}
+            </div>
+        </div>
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                <h4 class="panel-title">Project Details</h4>
+            </div>
+            <div class="panel-body">
+                <ul class="list-group">
+                    <li class="list-group-item col-sm-6">
+                        <b>Project Name:</b> {{ project.project_name}}
```

Comment:
Same note about bolding. As you've used `<b>` for a lot of bolding on different elements, I'd suggest just creating a `bold` class with `font-weight: bold` and putting it on all of the existing elements, and replacing any `<b>` with `<span class="bold">`